### PR TITLE
Support Details Date/Time in UTC

### PR DESCRIPTION
**Pull request by rayking29**
# Fixes [AB#15535](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/15535)

## Support Details Date/Time in UTC
- added ToLocalTime() call for CreatedDateTime and LastLoginDateTime

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## UI Changes

Screenshot 2023-05-09 at 2 30 22 PM

The time in the database (UTC) - note the createddatetime is converted to PST and the lastlogindatetime is converted to PDT
Screenshot 2023-05-09 at 2 30 12 PM

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by mk08sh — COMMENTED** [review]
👍

File: Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs [hunk 1]
```diff
@@ -113,6 +117,18 @@ private void RetrievePatientData()
             {
                 this.RetrieveMessagingVerifications();
             }
+
+            DateTime? patientProfileCreatedDateTime = this.Patient?.ProfileCreatedDateTime;
+            if (patientProfileCreatedDateTime != null)
+            {
+                this.ProfileCreatedDateTime = patientProfileCreatedDateTime.Value.ToLocalTime();
```
Comment (BrianMaki) [inline Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs:124]: Lines 124 and 130.  Why are we using ToLocalTIme() to convert to PST.  Are we not supposed to use the Timezone?  

Screenshot 2023-05-09 at 4 26 20 PM

**Review by BrianMaki — CHANGES_REQUESTED** [review]
Please see comment.

File: Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs (hunk 1 above)
Comment (rayking29) [reply Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs]: Fixed

**Review by MikeLyttle — APPROVED** [review]

**Review by BrianMaki — APPROVED** [review]

**Pull request merged**